new.css

```diff
@@ -213,14 +214,14 @@ input[type="reset"],
 input[type="button"] {
 	font-size: 1rem;
 	display: inline-block;
-	padding: 6px 12px;
+	padding: .5rem .75rem; /* updated pixel values to rem */
 	text-align: center;
 	text-decoration: none;
 	white-space: nowrap;
 	background: var(--nc-lk-1);
 	color: var(--nc-lk-tx);
 	border: 0;
-	border-radius: 4px;
+	border-radius: .5rem; /* updated pixel values to rem */
```

Comment:
Check the preview deploy, this doesn't look right.
Should be `.25rem` to match how it was, otherwise this is just an arbitrary change from original intent

Reply:
Yes that would be better, will update this